pkgs/development/interpreters/python/default.nix

```diff
@@ -198,6 +198,7 @@ in {
     inherit passthruFun;
     pythonAttr = "python3Minimal";
     # strip down that python version as much as possible
+    bash = bashUninteractive;
```

Comment:
```suggestion
    bash = bashNoninteractive;
```

Reply:
Of course, this applies to all other instances as well.

Reply:
i already upvoted @dnadlinger's comment which meant i was going to do it.

Reply:
Thank you, but why CamelCase? Noninteractive is one word.

Reply:
Thank you, but why CamelCase? Noninteractive is one word.